web/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from "react";
+
+import {
+  Table as MuiTable,
+  TableContainer,
+  TableHead,
+  TableBody,
+  Paper,
+} from "@mui/material";
+
+import { TableCell } from "./TableCell";
+import { TableRow } from "./TableRow";
+
+interface Props {
+  children: ReactNode;
+  dataTestId?: string;
```

Comment:
can't just extend TableContainerProps to include data-testId and all the rest props?